desktop/plugins/network/ManageMockResponsePanel.tsx

```diff
@@ -248,11 +259,13 @@ export function ManageMockResponsePanel(props: Props) {
       </LeftPanel>
       <RightPanel>
         {selectedId && props.routes.hasOwnProperty(selectedId) && (
+          <Layout.ScrollContainer vertical style={{height: 500}}>
```

Comment:
Why did you need a ScrollContainer here? To fix the header pane? In that case I'd move the scrolling inside the tab, so that the tab switcher stays always visible.

Reply:
This ScrollContainer can be removed.  Done.